gapic/src/main/com/google/gapid/models/Settings.java

```diff
@@ -73,7 +73,9 @@
           .setCheckForUpdates(true))
       .setUi(SettingsProto.UI.newBuilder()
           .setPerfetto(SettingsProto.UI.Perfetto.newBuilder()
-              .setDrawerHeight(250)))
+              .setDrawerHeight(250))
+          .setFramebufferPicker(SettingsProto.UI.FramebufferPicker.newBuilder()
+              .setEnabled(true)))
```

Comment:
Would it be better if we hide the feature for a while by setting the default toggle to false here? When we test the PR we can set it to true locally.